t/secrets.t

```diff
@@ -245,6 +245,29 @@ EOF
 	have_secret "$v/auto-generated-certs-a/ca:certificate";
 	my $ca = secret "$v/auto-generated-certs-a/ca:certificate";
 
+	sub get_cert_validity {
+		use Time::Piece;
```

Comment:
Move this to the top of the test file (if we have to keep Time::Piece; see below)